References/Resources.md

```diff
@@ -13,7 +13,10 @@
 - <https://developer.mozilla.org/en-US/docs/Web/JavaScript#reference>
 
 ### Markdown
-- <https://usu.instructure.com/courses/715206/discussion_topics/2551171>
+- ~~<https://usu.instructure.com/courses/715206/discussion_topics/2551171>~~
```

Comment:
```suggestion
- ~~<https://github.com/orgs/USU-ITLS-x265-23-summer/discussions>~~
```

Reply:
I think the old link points to an old course which we do not have access to. Since we are using GitHub's discussion board, I wonder if this should be the actual link.